### PR TITLE
New Crowdin updates

### PR DESCRIPTION
**Pull request by jjw24**

### CODE REVIEW

**Comment by jjw24** [comment]
Let's Leave this for after 1.19.1. New commits from Crowdin has reverted some language translations back to English due to minor grammar updates.

**Review by VictoriousRaptor — DISMISSED** [review]

File: Plugins/Flow.Launcher.Plugin.Explorer/Languages/uk-UA.xaml [hunk 1]
```diff
@@ -161,5 +161,5 @@
     <system:String x:Key="plugin_explorer_native_context_menu_header">Рідне контекстне меню</system:String>
     <system:String x:Key="plugin_explorer_native_context_menu_display_context_menu">Відображати рідне контекстне меню (експериментально)</system:String>
     <system:String x:Key="plugin_explorer_native_context_menu_include_patterns_guide">Нижче ви можете вказати елементи, які хочете включити до контекстного меню, вони можуть бути частковими (наприклад, «шир пера») або повними («Відкрити за допомогою»).</system:String>
-    <system:String x:Key="plugin_explorer_native_context_menu_exclude_patterns_guide">Нижче ви можете вказати елементи, які хочете виключити з контекстного меню, вони можуть бути частковими (наприклад, «шир пера») або повними («Відкрити за допомогою»)</system:String>
+    <system:String x:Key="plugin_explorer_native_context_menu_exclude_patterns_guide">Below you can specify items you want to exclude from context menu, they can be partial (e.g. 'pen wit') or complete ('Open with').</system:String>
```
Comment (jjw24) [inline Plugins/Flow.Launcher.Plugin.Explorer/Languages/uk-UA.xaml:164]:
```suggestion
    <system:String x:Key="plugin_explorer_native_context_menu_exclude_patterns_guide">Нижче ви можете вказати елементи, які хочете виключити з контекстного меню, вони можуть бути частковими (наприклад, «шир пера») або повними («Відкрити за допомогою»).</system:String>

```

File: Plugins/Flow.Launcher.Plugin.Explorer/Languages/zh-cn.xaml [hunk 2]
```diff
@@ -161,5 +161,5 @@
     <system:String x:Key="plugin_explorer_native_context_menu_header">本机上下文菜单</system:String>
     <system:String x:Key="plugin_explorer_native_context_menu_display_context_menu">显示本机上下文菜单（实验性）</system:String>
     <system:String x:Key="plugin_explorer_native_context_menu_include_patterns_guide">您可以在下面指定想要包含在上下文菜单中的项目，它们可以是部分的（例如“pen wit”）或完整的（“打开方式”）。</system:String>
-    <system:String x:Key="plugin_explorer_native_context_menu_exclude_patterns_guide">您可以在下面指定要从上下文菜单中排除的项目，它们可以是部分的（例如“pen wit”）或完整的（“打开方式”）。</system:String>
+    <system:String x:Key="plugin_explorer_native_context_menu_exclude_patterns_guide">Below you can specify items you want to exclude from context menu, they can be partial (e.g. 'pen wit') or complete ('Open with').</system:String>
```
Comment (jjw24) [inline Plugins/Flow.Launcher.Plugin.Explorer/Languages/zh-cn.xaml:164]:
```suggestion
    <system:String x:Key="plugin_explorer_native_context_menu_exclude_patterns_guide">您可以在下面指定要从上下文菜单中排除的项目，它们可以是部分的（例如“pen wit”）或完整的（“打开方式”）。</system:String>

```

File: Plugins/Flow.Launcher.Plugin.Program/Languages/cs.xaml [hunk 3]
```diff
@@ -82,7 +82,7 @@
 
     <system:String x:Key="flowlauncher_plugin_program_customizedexplorer">Vlastní Průzkumník</system:String>
     <system:String x:Key="flowlauncher_plugin_program_args">Arg</system:String>
-    <system:String x:Key="flowlauncher_plugin_program_tooltip_customizedexplorer">Umístění úvodní složky můžete upravit vložením proměnných prostředí, které chcete použít. Dostupnost proměnných prostředí můžete otestovat pomocí příkazového řádku.</system:String>
+    <system:String x:Key="flowlauncher_plugin_program_tooltip_customizedexplorer">You can customize the explorer used for opening the container folder by inputing the Environmental Variable of the explorer you want to use. It will be useful to use CMD to test whether the Environmental Variable is available.</system:String>
```
Comment (jjw24) [inline Plugins/Flow.Launcher.Plugin.Program/Languages/cs.xaml:85]:
```suggestion
    <system:String x:Key="flowlauncher_plugin_program_tooltip_customizedexplorer">Umístění úvodní složky můžete upravit vložením proměnných prostředí, které chcete použít. Dostupnost proměnných prostředí můžete otestovat pomocí příkazového řádku.</system:String>

```

File: Plugins/Flow.Launcher.Plugin.Program/Languages/ru.xaml [hunk 4]
```diff
@@ -82,7 +82,7 @@
 
     <system:String x:Key="flowlauncher_plugin_program_customizedexplorer">Настраиваемый проводник</system:String>
     <system:String x:Key="flowlauncher_plugin_program_args">Аргументы</system:String>
-    <system:String x:Key="flowlauncher_plugin_program_tooltip_customizedexplorer">Вы можете настроить проводник, используемый для открытия папки контейнера, введя переменную окружения проводника, который вы хотите использовать. Будет полезно использовать командную строку для проверки доступности переменной среды.</system:String>
+    <system:String x:Key="flowlauncher_plugin_program_tooltip_customizedexplorer">You can customize the explorer used for opening the container folder by inputing the Environmental Variable of the explorer you want to use. It will be useful to use CMD to test whether the Environmental Variable is available.</system:String>
```
Comment (jjw24) [inline Plugins/Flow.Launcher.Plugin.Program/Languages/ru.xaml:85]:
```suggestion
    <system:String x:Key="flowlauncher_plugin_program_tooltip_customizedexplorer">Вы можете настроить проводник, используемый для открытия папки контейнера, введя переменную окружения проводника, который вы хотите использовать. Будет полезно использовать командную строку для проверки доступности переменной среды.</system:String>

```

File: Plugins/Flow.Launcher.Plugin.Program/Languages/uk-UA.xaml [hunk 5]
```diff
@@ -82,7 +82,7 @@
 
     <system:String x:Key="flowlauncher_plugin_program_customizedexplorer">Кастомізований провідник</system:String>
     <system:String x:Key="flowlauncher_plugin_program_args">Аргументи</system:String>
-    <system:String x:Key="flowlauncher_plugin_program_tooltip_customizedexplorer">Ви можете налаштувати провідник, який використовується для відкриття теки контейнера, ввівши змінну середовища провідника, який ви хочете використовувати. Буде корисно скористатися командою CMD для перевірки доступності змінної середовища.</system:String>
+    <system:String x:Key="flowlauncher_plugin_program_tooltip_customizedexplorer">You can customize the explorer used for opening the container folder by inputing the Environmental Variable of the explorer you want to use. It will be useful to use CMD to test whether the Environmental Variable is available.</system:String>
```
Comment (jjw24) [inline Plugins/Flow.Launcher.Plugin.Program/Languages/uk-UA.xaml:85]:
```suggestion
    <system:String x:Key="flowlauncher_plugin_program_tooltip_customizedexplorer">Ви можете налаштувати провідник, який використовується для відкриття теки контейнера, ввівши змінну середовища провідника, який ви хочете використовувати. Буде корисно скористатися командою CMD для перевірки доступності змінної середовища.</system:String>

```

File: Plugins/Flow.Launcher.Plugin.Program/Languages/vi.xaml [hunk 6]
```diff
@@ -82,7 +82,7 @@
 
     <system:String x:Key="flowlauncher_plugin_program_customizedexplorer">Trình khám phá tùy chỉnh</system:String>
     <system:String x:Key="flowlauncher_plugin_program_args">Đối số</system:String>
-    <system:String x:Key="flowlauncher_plugin_program_tooltip_customizedexplorer">Bạn có thể tùy chỉnh trình thám hiểm được sử dụng để mở thư mục vùng chứa bằng cách nhập Biến môi trường của trình khám phá mà bạn muốn sử dụng. Sẽ rất hữu ích khi sử dụng CMD để kiểm tra xem Biến môi trường có sẵn hay không.</system:String>
+    <system:String x:Key="flowlauncher_plugin_program_tooltip_customizedexplorer">You can customize the explorer used for opening the container folder by inputing the Environmental Variable of the explorer you want to use. It will be useful to use CMD to test whether the Environmental Variable is available.</system:String>
```
Comment (jjw24) [inline Plugins/Flow.Launcher.Plugin.Program/Languages/vi.xaml:85]:
```suggestion
    <system:String x:Key="flowlauncher_plugin_program_tooltip_customizedexplorer">Bạn có thể tùy chỉnh trình thám hiểm được sử dụng để mở thư mục vùng chứa bằng cách nhập Biến môi trường của trình khám phá mà bạn muốn sử dụng. Sẽ rất hữu ích khi sử dụng CMD để kiểm tra xem Biến môi trường có sẵn hay không.</system:String>

```

File: Plugins/Flow.Launcher.Plugin.Program/Languages/zh-cn.xaml [hunk 7]
```diff
@@ -82,7 +82,7 @@
 
     <system:String x:Key="flowlauncher_plugin_program_customizedexplorer">自定义资源管理器</system:String>
     <system:String x:Key="flowlauncher_plugin_program_args">参数</system:String>
-    <system:String x:Key="flowlauncher_plugin_program_tooltip_customizedexplorer">您可以通过输入要使用的资源管理器的环境变量来自定义用于打开文件夹的资源管理器。 使用CMD来测试环境变量是否可用。</system:String>
+    <system:String x:Key="flowlauncher_plugin_program_tooltip_customizedexplorer">You can customize the explorer used for opening the container folder by inputing the Environmental Variable of the explorer you want to use. It will be useful to use CMD to test whether the Environmental Variable is available.</system:String>
```
Comment (jjw24) [inline Plugins/Flow.Launcher.Plugin.Program/Languages/zh-cn.xaml:85]:
```suggestion
    <system:String x:Key="flowlauncher_plugin_program_tooltip_customizedexplorer">您可以通过输入要使用的资源管理器的环境变量来自定义用于打开文件夹的资源管理器。 使用CMD来测试环境变量是否可用。</system:String>

```

**Comment by jjw24** [comment]
LGTM

**Review by onesounds — APPROVED** [review]

**Pull request merged**